orderapi/src/main/java/fi/hel/verkkokauppa/order/service/accounting/RefundAccountingService.java

```diff
@@ -0,0 +1,36 @@
+package fi.hel.verkkokauppa.order.service.accounting;
+
+import fi.hel.verkkokauppa.order.model.accounting.RefundAccounting;
+import fi.hel.verkkokauppa.order.repository.jpa.RefundAccountingRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class RefundAccountingService {
+
+    private Logger log = LoggerFactory.getLogger(RefundAccountingService.class);
```

Comment:
This logger can be done with class annotation `@Slf4j`

Reply:
done